qutebrowser/browser/commands.py

```diff
@@ -1847,3 +1847,27 @@ def debug_clear_ssl_errors(self):
         """Clear remembered SSL error answers."""
         nam = self._current_widget().page().networkAccessManager()
         nam.clear_all_ssl_errors()
+
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    def edit_url(self, url=None, bg=False, tab=False, window=False, count=None):
+        """Navigate to a url formed in an external editor.
+
+        The editor which should be launched can be configured via the
+        `general -> editor` config option.
+
+        Args:
+            url: URL to edit; defaults to the current page url.
+            bg: Open in a new background tab.
+            tab: Open in a new tab.
+            window: Open in a new window.
+            count: The tab index to open the URL in, or None.
+        """
+        self._editor = editor.ExternalEditor(
+            self._win_id, self._tabbed_browser)
+
+        # Passthrough for openurl args (e.g. -t, -b, -w)
+        self._editor.editing_finished.connect(
+            functools.partial(self.openurl,
+            bg = bg, tab = tab, window = window, count = count))
```

Comment:
Please remove the spaces around `=` here, as [per pep8](https://www.python.org/dev/peps/pep-0008/#other-recommendations) (as they are keyword arguments, not assignments)